lib/connectionContext.ts

```diff
@@ -12,7 +13,9 @@ import { TokenProvider } from "./auth/token";
 import { ManagementClient } from "./managementClient";
 import { CbsClient } from "./cbs";
 import { SasTokenProvider } from "./auth/sas";
+import { ClientOptions } from "./eventHubClient";
 
+const isBuffer = require("is-buffer");
```

Comment:
this is only really relevant for browser situations. otherwise you can just use [`Buffer.isBuffer()`](https://nodejs.org/api/all.html#buffer_class_method_buffer_isbuffer_obj)